packages/web-forms/src/components/FormLanguageMenu.vue

```diff
@@ -37,3 +37,26 @@ const handleLanguageChange = (event: LanguageDropdownChangeEvent) => {
 	</Dropdown>
 </template>
 
+<style scoped lang="scss">
+.p-dropdown.rounded {
+	border-radius: 30px;
+	border: none;
+	width: 160px;
+	color: #424242;
+
+	&.p-focus {
+		box-shadow: inset 0 0 0 1px var(--primary-500);
+	}
+
+	:deep(.p-dropdown-label) {
```

Comment:
Phew, for a minute there I thought there was a whole new CSS concept I missed. For those of us who are less familiar with Vue, `:deep` specifically breaches `scoped` styles applying the selector to children. Makes sense when applying styles into something opaque like components provided by PrimeVue.